linode/backup/framework_datasource.go

```diff
@@ -33,12 +32,19 @@ func (d *DataSource) Read(
 	client := d.Meta.Client
 
 	var linodeId types.Int64
```

Comment:
This `linodeId` can be removed in favor of `linodeID` :+1:

Reply:
Just fixed it. Thanks!